src/i_winmusic.c

```diff
@@ -19,33 +20,78 @@
 
 #include <windows.h>
 #include <mmsystem.h>
-#include <math.h>
+#include <mmreg.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <math.h>
 
 #include "doomtype.h"
+#include "i_sound.h"
+#include "i_system.h"
 #include "m_misc.h"
+#include "memio.h"
+#include "mus2mid.h"
 #include "midifile.h"
-#include "i_sound.h"
-#include "i_winmusic.h"
+#include "midifallback.h"
 
+char *winmm_midi_device = NULL;
+int winmm_reverb_level = -1;
+int winmm_chorus_level = -1;
+
+enum
+{
+    RESET_TYPE_NONE,
+    RESET_TYPE_GS,
+    RESET_TYPE_GM,
+    RESET_TYPE_GM2,
+    RESET_TYPE_XG,
+};
 
-#define BETWEEN(l,u,x) (((l)>(x))?(l):((x)>(u))?(u):(x))
+int winmm_reset_type = RESET_TYPE_GS;
+int winmm_reset_delay = 0;
 
-#define REVERB_MIN 0
-#define REVERB_MAX 127
-#define CHORUS_MIN 0
-#define CHORUS_MAX 127
+static byte gs_reset[] = {
```

Comment:
These arrays should be `const`

Reply:
Fixed in https://github.com/chocolate-doom/chocolate-doom/pull/1558/commits/2ce4fbc9d7bcd174b428073168f101fe92ae7178